test/xtypes/XTypesTests.cpp

```diff
@@ -1121,189 +1120,6 @@ TEST_F(xtypestests, DISABLED_TypeIdentifierNamesIgnored)
     pub.waitDiscovery(true, 3);
 }
 
-/**** TYPE INFORMATION ****/
-
-/*
- * Both endpoints share the same type using XCDR1 so they must match.
- * This represents classical match.
- */
-TEST_F(xtypestests, TypeInformationSameType)
```

Comment:
Please, ensure that these same test cases are tested in the blackbox/communication tests.

Reply:
Will create a PR with tests